src/i18n/en.json

```diff
@@ -1,188 +1,186 @@
-    "sendAgainHelpText": "Approval email has been re-sent to ",
-    "buttonText": "Resend the membership request",
-    "linkToShowSentData": "Display your application"
-  },
-  "approval": {
-    "title": "Please approve membership of Nuta",
-    "formExplanationText_1": "has sent you Nuta membership request for approval. By approving the membership,",
-    "formExplanationText_2": "could use services for youth and get benefits. In case of incorrect data, ask youth to edit info and send a new request.",
-    "basicInfo": "Basic info",
-    "name": "Name",
-    "profile": "Profile",
-    "address": "Home address",
-    "phone": "Phone",
-    "birthDate": "Birthdate",
-    "addInfo": "Additional info",
-    "schoolInfo": "School and class",
-    "languagesAtHome": "Languages at home",
-    "approverAcceptance": "Permissions to be approved",
-    "photoUsageApproved": "Photo usage approval",
-    "photoUsageApprovedText": "Giving permission to use photos of my child for Helsinki internal and external media purposes.",
-    "photoUsageApprovedYes": "Yes",
-    "photoUsageApprovedNo": "No",
-    "approverInfo": "Approver info",
-    "approverInfoText": "Guardian may be contacted concerning the child.",
-    "approverFirstName": "Firstname",
-    "approverLastName": "Lastname",
-    "approverEmail": "Email",
-    "approveTermsText_1": "I accept",
-    "approveTermsText_link": " service terms",
-    "approveTermsText_2": ". I can be contacted via address I have given.",
-    "approveButton": "Approve membership",
-    "approvedTitle": "Nuta membership approved!",
-    "approvedMessage": "Thank you for confirming youth service membership.",
-    "approvedLink": "Application is already approved / link is defective"
-  },
-  "validation": {
-    "tooLong": "Too long",
-    "tooShort": "Too short",
-    "required": "Required",
-    "phoneMin": "Min length six characters",
-    "email": "Invalid email"
-  },
-  "footer": {
-    "copyright": "Copyright {{year}}",
-    "reserveRights": "All rights reserved",
-    "privacy": "Privacy policy",
-    "accessibility": "Accessibility document",
-    "terms": "Terms of Service",
-    "feedback": "Give feedback",
-    "feedbackLink": "https://www.hel.fi/helsinki/en/administration/participate/feedback"
+    "schoolClass": "School class",
+    "schoolName": "School name",
+    "sendButton": "Save and send approval request",
+    "title": "Please fill in your information"
   },
+  "sanityCheck": "youth-profile.en",
   "tos": {
+    "message1": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
```

Comment:
Is there a Jira-issue for turning this from lorem-ipsum to something? 
Could be fun to launch the project with this text, and have a bet going between the team members about how many days it will take before anyone notices :-)

Reply:
There is [this one](https://helsinkisolutionoffice.atlassian.net/browse/OM-618#icft=OM-618) (it is already done though). I'll double check with Anniiina.

Reply:
Seems there is issue about this. [Link](https://helsinkisolutionoffice.atlassian.net/browse/YM-170)